cluster/gce/manifests/etcd.manifest

```diff
@@ -59,7 +54,62 @@
       { "name": "ETCD_HOSTNAME",
         "value": "{{ hostname }}"
       }
-        ],
+    ],
+    "volumeMounts": [
+      { "name": "varetcd",
+        "mountPath": "/var/etcd",
+        "readOnly": false
+      },
+      { "name": "varlogetcd",
+        "mountPath": "/var/log/etcd{{ suffix }}.log",
+        "readOnly": false
+      },
+      { "name": "etc",
+        "mountPath": "/etc/srv/kubernetes",
+        "readOnly": false
+      }
+    ]
+    },
+    {
+    "name": "etcd{{ suffix }}-init-container-2",
+    "image": "k8s.gcr.io/debian-base:v2.0.0",
+    "command": [
+      "/bin/sh",
+      "-c",
+      "chown -R {{runAsUser}}:{{runAsGroup}} /var/etcd/"
```

Comment:
if this fails, what happens to the pod? does it retry? permanently fail?

Reply:
It will retry. See [this](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#understanding-init-containers) for details.